src/stdlib/string_utils.rs

```diff
@@ -0,0 +1,9 @@
+use crate::prelude::{Value, ValueError, VrlValueConvert};
+
+pub(crate) fn convert_to_string(value: Value, case_sensitive: bool) -> Result<String, ValueError> {
```

Comment:
`case_sensitive` doesn't really make sense here. It's not clear if it's converting to lowercase or uppercase. Maybe just call it `to_lowercase: bool`?